README.md

```diff
@@ -101,13 +101,13 @@ There are GH actions defined to actually deploy the serverless and wrangler part
 
 optional values are *italic* (i.e. will be using defaults if not set)
 
-## [API Docs](https://github.com/tyvdh/stellar-tss/wiki)
+## [API Docs](https://tyvdh.github.io/stellar-turrets/)
 
 ## Disclaimer
-This is alpha software representing a reference implementation for the [TSS protocol](https://tss.stellar.org/).
+This is alpha software representing a reference implementation for the [Stellar Turrets protocol](https://tss.stellar.org/).
```

Comment:
This is definitely a decision above my paygrade, but would it make sense at some point to replace `tss.stellar.org` with `turrets.stellar.org` or something?

Reply:
It definitely would. I have an outstanding ticket with our ops team so just waiting on that